Aws/S3/Commands/Multipart.hs

```diff
@@ -131,12 +131,13 @@ data UploadPart = UploadPart {
   , upContentMD5 :: Maybe (Digest MD5)
   , upServerSideEncryption :: Maybe ServerSideEncryption
   , upRequestBody  :: HTTP.RequestBody
+  , upExpect100Continue :: Bool -- ^ Note: Requires http-client >= 0.4.10
```

Comment:
Isn't this probably behavior that is always positive for uploads? If so, why not just enable it based on the http-client version?

Hmm but then the question is also if all S3 clones support it...